lib/ReplaceSource.js

```diff
@@ -149,52 +137,208 @@ class ReplaceSource extends Source {
 		return map;
 	}
 
-	_replacementToSourceNode(oldNode, newString) {
-		var map = oldNode.toStringWithSourceMap({
-			file: "?"
-		}).map;
-		var original = new SourceMapConsumer(map.toJSON()).originalPositionFor({
-			line: 1,
-			column: 0
+	_splitString(str, position) {
+		return position <= 0 ? ["", str] : [str.substr(0, position), str.substr(position)];
+	}
+
+	_sortReplacementsAscending() {
+		this.replacements.sort(function(a, b) {
```

Comment:
Could you define the sort function outside of the class?

Reply:
@brentwilton bump

Reply:
Moved outside of class as requested.